configs/defs/generic_linux.textproto

```diff
@@ -18616,7 +18860,48 @@ benchmark_configs: {
       "                    match_type: ALL_MATCH_ANY_ORDER"
       "                    match_criteria:{"
       "                        filter_regex: \".*\\\\(\\\\w+\\\\)\""
-      "                        expected_regex: \".*\\\\(enforce|complain\\\\)\""
+      "                        expected_regex: \".*\\\\(complain\\\\)\""
```

Comment:
Why just "complain"? The check says it should either be enforce or complain mode